specutils/spectra/spectrum1d.py

```diff
@@ -50,10 +50,15 @@ class Spectrum1D(OneDSpectrumMixin, NDDataRef):
         See `redshift` for more information.
     radial_velocity
         See `radial_velocity` for more information.
+    bin_specification : str
+        Either "edges" or "centers" to indicate whether the `spectral_axis`
+        values represent edges of the wavelength bin, or centers of the bin.
     uncertainty : `~astropy.nddata.NDUncertainty`
         Contains uncertainty information along with propagation rules for
         spectrum arithmetic. Can take a unit, but if none is given, will use
         the unit defined in the flux.
+    mask : `~numpy.ndarray`-like and bool
```

Comment:
As I recall, it is *not* strictly necessary that this be a bool.  Rather it can be anything so long as "0"/"False" is "good".  That is, `mask` can either be a bool *or* a "DQ" array.